Server/Controllers/CashByCodeController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using MediatR;
+using Endava_Project.Server.Models;
+using Microsoft.AspNetCore.Identity;
+using Endava_Project.Shared;
+using Endava_Project.Server.Application.CashByCodeMethods.Query;
+using Endava_Project.Server.Application.CashByCodeMethods.Command;
+using AutoMapper;
+
+namespace Endava_Project.Server.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CashByCodeController : ControllerBase
+    {
+        private readonly UserManager<ApplicationUser> userManager;
+        private readonly IMediator mediator;
+
+        public CashByCodeController(UserManager<ApplicationUser> userManager, IMediator mediator)
+        {
+            this.userManager = userManager;
+            this.mediator = mediator;
+        }
+
+        [HttpGet]
+        [Route("getall")]
+        public async Task<List<CashByCodeDto>> GetAllCashByCode()
+        {
+            var query = new GetAllCashByCodeQuery
+            {
+                UserId = userManager.GetUserId(User)
+            };
+
+            var cashByCodeList = await mediator.Send(query);
+            var list = new List<CashByCodeDto>();
+            var config = new MapperConfiguration(cfg => cfg.CreateMap<CashByCode, CashByCodeDto>());
```

Comment:
As a side note, it's possible to create a mapping [Profile](https://docs.automapper.org/en/stable/Configuration.html). And [register](https://docs.automapper.org/en/stable/Dependency-injection.html) this profile in the Dependency Injection container.
Therefore, your controller will simply receive an IMapper and you won't need to manually configure it in each action.